sources/web/datalab/polymer/components/file-browser/file-browser.ts

```diff
@@ -158,8 +158,15 @@ class FileBrowserElement extends Polymer.Element implements DatalabPageElement {
 
     this._apiManager = ApiManagerFactory.getInstance();
 
+    // Allow forcing an file manager type.
```

Comment:
Is this supposed to be "any file manager type"?

Reply:
Done.